lim_test.py

```diff
@@ -0,0 +1,259 @@
+'''tests for the longintmath.py module'''
+import unittest
+import longintmath as lim
+
+class TestLIMFunc(unittest.TestCase):
+    '''tests for the long_int_(mathfunction) functions'''
+    def setUp(self):
+        self.long_big = 10**1000
```

Comment:
since these are not mutated you do not need them in the setup and tear down. Those are only for mutable state.